3.5/security-security-options.md

```diff
@@ -40,67 +40,134 @@ an overview of the relevant options.
     --javascript.startup-options-blacklist "^javascript\."
-    --javascript.startup-options-blacklist "endpoint"
+    --javascript.startup-options-blacklist "^endpoint$"
 
-will resolve internally to the following regular expressions:
+These sets will resolve internally to the following regular expressions:
 
 ```
 --javascript.startup-options-whitelist = "^server\.|^log\."
 --javascript.startup-options-blacklist = "^javascript\.|endpoint"
 ```
 
-Access to directories and files from JavaScript operations is only 
-controlled via a whitelist, which can be specified via the startup
-option `--javascript.files-whitelist`.
+Invoking an arangosh with these options will hide the blacklisted commandline
+options from the output of: 
+
+```js
+require('internal').options()
+```
+
+… and an exception will be thrown when trying to access items that are masked
+in the same way as if they weren't there in first place.
+
+#### File access
+
+In contrast to other areas, access to directories and files from JavaScript
+operations is only controlled via a whitelist, which can be specified via the
+startup option `--javascript.files-whitelist`. Thus any files or directories
+not matching the whitelist will be inaccessible from JavaScript filesystem
+functions.
 
 For example, when using the following startup options
-    
-    --javascript.startup-options-whitelist "^/etc/required/"
-    --javascript.startup-options-whitelist "^/etc/mtab/"
 
-all files in the directories `/etc/required` and `/etc/mtab` plus their
-subdirectories will be accessible, while access to files in any other directories 
+    --javascript.files-whitelist "^/etc/required/"
+    --javascript.files-whitelist "^/etc/mtab/"
+    --javascript.files-whitelist "^/etc/issue$"
+
+The file `/etc/issue` will be allowed to accessed and all files in the directories
+`/etc/required` and `/etc/mtab` plus their subdirectories will be accessible,
+while access to files in any other directories 
 will be disallowed from JavaScript operations, with the following exceptions:
 
 - ArangoDB's temporary directory: JavaScript code is given access to this
   directory for storing temporary files. The temporary directory location 
   can be specified explicitly via the `--temp.path` option at startup. 
   If the option is not specified, ArangoDB will automatically use a subdirectory 
-  of the system's temporary directory).
+  of the system's temporary directory.
 - ArangoDB's own JavaScript code, shipped with the ArangoDB release packages.
   Files in this directory and its subdirectories will be readable for JavaScript
   code running in ArangoDB. The exact path can be specified by the startup option 
   `--javascript.startup-directory`.
 
+#### Endpoint access
+
+The endpoint black/white listing limits access to external HTTP resources. 
+In contrast to the URLs specified in the JavaScript code, the filters have
+to be specified in the ArangoDB endpoints notation: 
+
+- http:// => tcp://
+- https:// => ssl://
```

Comment:
What about the other aliases we have (at least in client tools)?
`http+tcp`, `http+ssl`, `srv` (Windows only)...

Reply:
no, needs to be the propper internal representation as documented. Should I add these to that list?